src/AppBundle/Resources/views/menu.html.twig

```diff
@@ -11,7 +11,7 @@
       <a class="navbar-brand" href="{{ path('homepage') }}">CoopCycle</a>
     </div>
 
-    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```

Comment:
If you do that, you also need to modify the `data-target` of the toggle element.

Reply:
Aggreed, I don't see why you need to change the ID here (I possibly misguided you on this).